vscode/package.json

```diff
@@ -59,6 +121,41 @@
         "category": "Debug",
         "enablement": "!inDebugMode",
         "icon": "$(play)"
+      },
+      {
+        "command": "qsharp-vscode.workspacesRefresh",
```

Comment:
Should these commands be enabled conditional on `AzureQuantumIntegration` as well?

Reply:
I should probably just remove that setting and have Azure integration always on, now that we plan to have it ready by preview.

Reply:
Removed the AzureQuantumIntegration option (it's not optional now)